py2dash/app_makers.py

```diff
@@ -2,15 +2,25 @@
 import dash_core_components as dcc
 import dash_html_components as html
 import flask
-from dash.dependencies import Output, Input
+from dash.dependencies import Output, Input, State
 
 from py2dash import component_makers as cm
 
 
-def dispatch_func_to_app(app, func):
+def dispatch_func_to_app(app, func, execute_on_change=False):
```

Comment:
Compare the use of the execute_on_change argument to putting the argument as "VAR_KEYWORD" (e.g. **kwargs), or as the attribute of some config object. 

dispatch_func_to_app is a general function that will do it's job based on all kinds of specifications, of which execute_on_change is only one of them. This won't scale well, so we must think of the proper strategy.

I'll make an issue to discuss this.